extensions/default/src/Components/SidePanelWithServices.tsx

```diff
@@ -83,18 +97,18 @@ const SidePanelWithServices = ({
     return () => {
       activatePanelSubscription.unsubscribe();
     };
-  }, [tabs, sidePanelOpen, panelService]);
+  }, [tabs, sidePanelOpen, panelService, closedManually]);
 
   return (
     <SidePanel
       {...props}
       side={side}
       tabs={tabs}
       activeTabIndex={activeTabIndex}
+      isExpanded={sidePanelOpen}
```

Comment:
if `isExpanded={sidePanelOpen}` can we name it isOpen then?

Reply:
I prefer to replace `sidePanelOpen` with `sidePanelExpanded` instead because in general the lingo is expanded/collapsed for resizable side panels.